activity_calendar/templates/activity_calendar/activity_moment_form_page.html

```diff
@@ -0,0 +1,58 @@
+{% extends 'core/base.html' %}
+
+{% load static %}
+{% load render_form %}
+{% load generic_field %}
+{% load activity_default_display_tag %}
+
+{% block content-frame-class %}
+    wideContentFrame
+{% endblock %}
+
+{% block content %}
+    <h2>Change specific activity moment</h2>
```

Comment:
There's a mismatch between the terms "activity moment" and "activity instance". (I'd suggest we use "activity instance" or even "occurrence")